launch/cob4_test_launch_node.launch

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```

Comment:
as you go with `ipa_` prefix in the code files...maybe you want to keep it here rather than use `cob4`
(maybe this is too nitpicky - just saying...)